tests/foreman/ui/test_org.py

```diff
@@ -554,27 +560,28 @@ def test_add_user(self, user_name):
 
         """
         strategy, value = common_locators['entity_deselect']
-        org_name = gen_string('alpha')
-        user = entities.User(
-            login=user_name,
-            firstname=user_name,
-            lastname=user_name,
-            password=gen_string('alpha')
-        ).create()
-        self.assertEqual(user.login, user_name)
         with Session(self.browser) as session:
-            make_org(session, org_name=org_name)
-            self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_users=[user_name])
-            self.org.search(org_name).click()
-            session.nav.click(tab_locators['context.tab_users'])
-            element = session.nav.wait_until_element((strategy,
-                                                      value % user_name))
-            self.assertIsNotNone(element)
+            for user_name in valid_users():
```

Comment:
Same comment as before about creating only one organization and then adding users. Not a blocker for this PR. Notice that this test could be improved by checking that: a) the new user has indeed been added to the organization; b) the total number of users is incremented by 1 for every iteration.